fuzz_test.go

```diff
@@ -504,3 +505,37 @@ func TestFuzz_SkipPattern(t *testing.T) {
 		return 5, true
 	})
 }
+
+func Test_charRange_choose(t *testing.T) {
+	// We run 10,000 iterations of charRange.choose and verify 'a', 'z' (edge cases) and 'm'
```

Comment:
Suggestion: make a fake rand, e.g.:

```
type alwaysLast struct {
  *rand.Rand
}

func (a alwaysLast) Int63n(n int64) int64 {
  return n-1
}
```

(you can imagine various extensions to this to test other edge cases)

Then you can test the edge cases instead of trying a bunch of times.

Reply:
done